nginx/ngx_js.c

```diff
@@ -4493,3 +4493,19 @@ ngx_js_queue_pop(ngx_js_queue_t *queue)
 
     return item;
 }
+
+
+ngx_int_t
+ngx_njs_string(njs_vm_t *vm, njs_value_t *value, ngx_str_t *str)
```

Comment:
Instead of introducing ngx_njs_string(), we should make ngx_js_string() with that prototype, which accepts ngx_str_t *str. and fix the code globally in `nginx/*`.